compute/admin_guide/install/system_requirements.adoc

```diff
@@ -434,14 +459,62 @@ ECS agent 1.61.0
 
 Docker 20.10.7
 
-|Kubernetes
+|Kubernetes with Docker
 |Docker Engine version: 20.10.14
 
 API version:1.41
 
 Go Version: go1.16.15
 
-Native Kubernetes 1.23.5 (containerd 1.5.11)
+|Kubernetes with containerd
+|Kubernetes 1.23.5 (containerd 1.5.11)
+
+|Lightweight Kubernetes (k3s)
+|k3s version: 1.23.8+k3s2 (containerd 1.5.13-k3s1)
+
+k3s version: v1.23.6+k3s1 (containerd 1.5.11-k3s2)
+
+k3s version: v1.23.8+k3s1 (containerd 1.5.13-k3s1)
+
+|GKE on ARM
```

Comment:
We need to specify that we support ARM on GCP as part of the "ARM Architecture Requirements" section (not just AWS graviton).
We need to add it to the release notes as well

Reply:
@MorShabiPAN Before adding the blurbs to the release notes, I wanted to ensure that we had the correct information here.
Thank you!

Reply:
I've added the line to the RNs and updated the table.